peter-lang-verlag.csl

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" page-range-format="expanded">
+  <info>
+    <title>Peter Lang Verlag</title>
+    <title-short>Peter Lang</title-short>
+    <id>http://www.zotero.org/styles/peter-lang-verlag</id>
+    <link href="http://www.zotero.org/styles/peter-lang-verlag" rel="self"/>
+    <author>
+      <name>J. R. Schmid</name>
+      <email>jrs+zotero@weitnahbei.de</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="anthropology"/>
+    <category field="communications"/>
+    <category field="generic-base"/>
+    <category field="sociology"/>
+    <category field="humanities"/>
+    <category field="history"/>
+    <category field="literature"/>
+    <category field="philosophy"/>
+    <category field="political_science"/>
+    <category field="social_science"/>
+    <category field="linguistics"/>
+    <summary>Zitierstil soweit möglich den Vorgaben des Verlags Peter Lang entsprechend</summary>
+    <updated>2017-05-11T14:13:23+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="author">
+    <names variable="author original-author" font-style="italic">
+      <name and="text" initialize-with=". " name-as-sort-order="all"/>
+    </names>
+  </macro>
+  <macro name="author-truncated">
+    <names variable="author original-author" font-style="italic">
+      <name and="text" initialize-with=". " name-as-sort-order="all"/>
+    </names>
+  </macro>
+  <macro name="container:book">
+    <group delimiter=", " prefix="In: ">
+      <names variable="collection-editor container-author editor editorial-director" suffix=" (ed.)">
+        <name form="short"/>
+      </names>
+      <text variable="container-title" form="short" strip-periods="false"/>
+      <text macro="place-and-year"/>
+      <text macro="location"/>
+    </group>
+  </macro>
+  <macro name="container:encyclopaedia">
+    <group delimiter=", " prefix="In: ">
+      <text variable="container-title" form="short"/>
+      <text macro="place-and-year"/>
+      <text macro="location"/>
+    </group>
+  </macro>
+  <macro name="container:journal">
+    <group delimiter=", " prefix="In: ">
+      <text variable="container-title" strip-periods="false"/>
+      <group>
+        <text variable="publisher-place" suffix=" "/>
+        <text variable="issue"/>
+        <text macro="year" prefix="(" suffix=")"/>
+        <text variable="volume"/>
+      </group>
+    </group>
+  </macro>
+  <macro name="title">
+    <text variable="title" form="short"/>
+  </macro>
+  <macro name="title-truncated">
```

Comment:
The style is not validating as you have this macro here that is actually then not used and CSL likes no unnecessary code. So either delete or use it (if you further build this style).
Also, see this discussion here regarding on how to use the shortened titles: https://forums.zotero.org/discussion/65629/csl-macro-to-abbreviate-strings#latest